Patches/DleksPatch.cs

```diff
@@ -45,8 +45,27 @@ public static void Postfix_AllMapIcons(GameStartManager __instance)
     {
         if (__instance == null) return;
 
+        if (GameStates.IsNormalGame && Main.NormalOptions.MapId == 3)
+        {
+            Main.NormalOptions.MapId = 0;
+            __instance.UpdateMapImage(MapNames.Skeld);
+
+            if (!Options.RandomMapsMode.GetBool())
+                CreateOptionsPickerPatch.SetDleks = true;
+        }
+        else if (GameStates.IsHideNSeek && Main.HideNSeekOptions.MapId == 3)
+        {
+            Main.HideNSeekOptions.MapId = 0;
+            __instance.UpdateMapImage(MapNames.Skeld);
+
+            if (!Options.RandomMapsMode.GetBool())
+                CreateOptionsPickerPatch.SetDleks = true;
+        }
```

Comment:
isn't this the exact same code jsut do a || statement in the thing above 😆

Reply:
nvm I am blind

Reply:
?

Reply:
forget about it I just didn't see something